docs/why_cumulusci.rst

```diff
@@ -115,27 +120,54 @@ There are some key differentiators to how CumulusCI works in comparison to Sales
+
+MetaCI
+------
+MetaCI is our custom CI app run on Heroku to automate the execution of builds using CumulusCI flows.  It is Salesforce aware and can handle burst capacity for builds by leveraging Heroku's scalability.
 
-* **Automated Release Notes**: CumulusCI's **release_beta** flow uses the **github_release_notes** task to automatically parse the bodies of pull requests merged since the last production release and generate combined release notes from the content.
-* **Bulk API Query/Load**: CumulusCI includes Python task classes allowing for the creation of multi-object relational data set mappings used to query data from a Salesforce org into a local sqlite database and insert that relational data into another Salesforce org.
-* **Dependency Management**: CumulusCI includes robust support for project dependencies including managed packages, unmanaged metadata, and references to other CumulusCI project repositories to dynamically and recursively inherit the referenced project's dependencies
-* **Apex Limit Reports for Tests**: CumulusCI's Apex test runner outputs a **test_results.json** file which includes the duration and Apex limits usage for each test method executed
-* **Update Admin Profile**: All CumulusCI flows run the **update_admin_profile** task to retrieve the Admin.profile from the target org after deploying the package or the package source, grant FLS permissions on all fields and classes, and deploy the updated profile.  This makes it easier to get up and running with a useable environment from a fresh scratch org.
-* **Push API**: Automate push upgrades of your product using the Push API and CumulusCI's built in tasks: **push_all**, **push_sandbox**, **push_trial**, and **push_qa**
-* **meta.xml File Management**: Unmanaged deploys automatically strip namespace, majorVersion, and minorVersion elements from the meta.xml file allowing CumulusCI's dependency management to handle your dependencies.  Also, the **meta_xml_dependencies** and **meta_xml_apiversion** tasks automate updating all local meta.xml files with the api_version specified in **cumulusci.yml** and the namespace, majorVersion, and minorVersion of the currently resolved dependencies.
-* **MetaCI**: MetaCI is our custom CI app run on Heroku to automate the execution of builds using CumulusCI flows.  It is Salesforce aware and can handle burst capacity for builds by leveraging Heroku's scalability.
+MetaDeploy
+----------
+MetaDeploy is our custom installer app run on Heroku to automate complex installations of managed package projects. It help us to provide our customers with a seemless install experience.
 
 Next Steps
 ==========
 
-Interested in trying it out?  Check out the CumulusCI :doc:`tutorial`
+Interested in trying it out?  Check out the CumulusCI :doc:`tutorial` or our :doc:`usage` screencasts.
```

Comment:
Can we pull in the link to the trailhead from the new docs? That's a better place to point people than the tutorial.